lib-php/PlaceLookup.php

```diff
@@ -300,10 +300,11 @@ public function lookup($aResults, $iMinRank = 0, $iMaxRank = 30)
                 $sSQL .= 'null::text AS names,';
             }
             $sSQL .= '  ST_x(geometry) AS lon, ST_y(geometry) AS lat,';
-            $sSQL .= '  (0.75-(rank_search::float/40)) AS importance, ';
+            $sSQL .= '  importance, ';
             $sSQL .= $this->addressImportanceSql('geometry', 'lp.parent_place_id');
             $sSQL .= '  null::text AS extra_place ';
-            $sSQL .= 'FROM location_postcode lp';
+            $sSQL .= 'FROM location_postcode lp ';
+            $sSQL .= 'JOIN placex USING (place_id)';
```

Comment:
Postcodes don't have an importance, as you correctly noticed. Let's keep the rank formula here.